src/main/haskell/kore/src/Kore/Parser/LexemeImpl.hs

```diff
@@ -121,9 +124,10 @@ skipWhitespace = L.space Parser.space1 (L.skipLineComment "//") blockComment
     blockComment = Parser.string "/*" >> void (manyTill commentBody (Parser.string "*/"))
     commentBody = Parser.anyChar <|> (eof >> fail "Unfinished comment.")
 
-koreKeywordsSet :: Trie.Trie ()
-koreKeywordsSet = Trie.fromList $ map (\s -> (Char8.pack s, ()))
-    ["module", "endmodule", "sort", "symbol", "alias", "axiom"]
+koreKeywordsSet :: HashSet Char8.ByteString
```

Comment:
Why replacing Trie by set?

Reply:
I think Trie is unnecessarily high-powered for short lists of short strings.

Reply:
The package that provides `Trie` (`bytestring-trie`) was removed from LTS Haskell because it is not maintained. `HashMap` and `HashSet` both have O(1) lookup; the only advantage of `Trie` is possible sharing between the keys. With so few keys, and such small keys, we don't get any sharing anyway.